tokio-util/tests/spawn_pinned.rs

```diff
@@ -1,5 +1,7 @@
 #![warn(rust_2018_idioms)]
 #![cfg(not(target_os = "wasi"))] // Wasi doesn't support threads
+// Blocked on https://github.com/rust-lang/miri/issues/3911
+#![cfg(not(miri))]
```

Comment:
Was this fixed?

Reply:
Yes, but I don't think the latest fix is synced to rustc yet.